@navikt/core/react/src/table/stories/table-expandable.stories.tsx

```diff
@@ -227,3 +230,72 @@ export const ExpandableOpen = () => {
     </Table>
   );
 };
+
+export const ClickableRow = {
+  render: () => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [row1, setOpen1] = useState(false);
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [row2, setOpen2] = useState(false);
+
+    return (
+      <Table zebraStripes>
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell>Navn</Table.HeaderCell>
+            <Table.HeaderCell>Info</Table.HeaderCell>
+            <Table.HeaderCell aria-hidden />
+          </Table.Row>
+        </Table.Header>
+        <Table.Body>
+          <Table.ExpandableRow
+            content={<div>placeholder</div>}
+            togglePlacement="right"
+            onOpenChange={setOpen1}
+            data-testid="row1"
+            open={row1}
+          >
+            <Table.DataCell>Ola</Table.DataCell>
+            <Table.DataCell>
+              <Button size="xsmall">Mer info</Button>
+            </Table.DataCell>
+          </Table.ExpandableRow>
+          <Table.ExpandableRow
+            content={<div>placeholder</div>}
+            togglePlacement="right"
+            onOpenChange={setOpen2}
+            data-testid="row2"
+            open={row2}
+          >
+            <Table.DataCell>Hans</Table.DataCell>
+            <Table.DataCell>
+              <Checkbox hideLabel size="small">
+                Sett
+              </Checkbox>
+            </Table.DataCell>
+          </Table.ExpandableRow>
+        </Table.Body>
+      </Table>
+    );
+  },
+  play: async ({ canvasElement, step }) => {
```

Comment:
Laget en interaktiv demo (kan ses i storybook) for et av problemene som har stoppet oss fra denne oppdateringen tidligere:

I tilfeller der man har andre interaktive elementer som eks lenker, checkboxer eller knapper ønsker vi at man ikke skal åpne raden ved klikk på dem. Dette kan kanskje også videreføres til at raden ikke skal åpnes hvis man ønsker å eks kopiere noe tekst, men ikke sikker på den.

Arbeidet stoppet litt opp med å finne en god løsning på dette, men kan hende alt man trenger er å sjekke `document.activeElement` 🤞 Vært helt mega å finne en god løsning på dette da jeg vet mange ønsker seg denne featuren 🙌

Reply:
Takk for eksemplet.
Eventet vil per default spre seg videre ja. Dersom de interaktive elementene skal hindre eventet til å gå videre til raden må det ansvaret være på dem. Jeg la til en `event.stopPropagation` i click-handlere på både `Button` og `Checkbox` i eksempelet.
Jeg kommenterte også ut "play" bitten slik at alerts ikke kjører med en gang.

Reply:
:blowfish: Update: Har lagt til en ny commit igjen nå, hvor jeg fjerner stopPropagation fra elementene i storyen, og istedet legger til logikk på selve ExpandableRow for å kun ekspande ved behov. Se linje `73-89`.

:mushroom: Det kan være en mer robust løsning som gjør at konsumentene ikke behøver å forholde seg til det.